documents/src/pages/elements/number-field.md

```diff
@@ -190,3 +207,37 @@ element.addEventListener('error-changed', (e) => {
   }
 });
 ```
+
+## Accessibility
+::a11y-intro::
+
+`ef-number-field` is assigned `role="textbox"`. States such as `disabled` or `read-only` are programmatically updated to match the element’s visual state. 
```

Comment:
```suggestion
`ef-number-field` is assigned `role="textbox"`. States such as `disabled` or `readonly` are programmatically updated to match the element’s visual state. 
```

I'm not sure how to put this but internal `input` has role of `spinbutton`.

Reply:
So why we assign role textbox at host but internal input is spinbutton? What screen reader going to see, textbox or spinbutton?

Reply:
It will always delegate focus to the internal input.

Reply:
Ok so it seems that host is not assigned with textbox.  I think we should change to say that it has `spinbutton` role.
Make sense?